README.md

```diff
@@ -2,7 +2,7 @@
 
 <img src="logo/coaler-removebg.png" width="150">
 
-Core Aligner
```

Comment:
Why renaming?

Reply:
hahah wtf I guess CLion did this when I renamed the "Core" class :facepalm: